static/app/views/organizationDetails/index.spec.tsx

```diff
@@ -15,12 +16,14 @@ jest.mock(
 );
 
 describe('OrganizationDetails', function () {
+  const {routerProps} = initializeOrg();
```

Comment:
You can also TestStubs.routeComponentProps() if that is all that you need. Initialize orgs initializes a lot of unnecessary things in this case

Reply:
good to know that :) will use it next time.